cli/src/api/templates/load-wasi-template.ts

```diff
@@ -24,8 +24,10 @@ const __wasi = new __WASI({
 const __emnapiContext = __emnapiGetDefaultContext()
 
 const __sharedMemory = new WebAssembly.Memory({
-  initial: 1024,
-  maximum: 10240,
+  // 1Gb
+  initial: 16384,
```

Comment:
The `initial` size means `new WebAssembly.Memory()` will allocate 1GB size `SharedArrayBuffer` immediately, I personally don't think most of normal wasm module need so much initial memory, it just grow within `maximum` on demand. If there are multiple napi-rs modules are using in a same web page or a StackBlitz Node.js worker, is this too waste?

Reply:
Good point. Let's make it configurableble